classes/User.php

```diff
@@ -92,6 +99,7 @@ class User extends Fighter {
 
     public $exp;
     public $staff_level;
+    public bool|StaffManager $staff_manager;
```

Comment:
Generally if a class is either present or not, I think nullable can be a nicer way to represent that, you can do `?StaffManager`